payara-micro:start NullPointerException

Attached is a reproducer.

Steps to reproduce:
- `mvn clean install`
- `mvn payara-micro:start -e`

Result: 

```
[ERROR] Failed to execute goal fish.payara.maven.plugins:payara-micro-maven-plugin:1.0.0-SNAPSHOT:start (default-cli) on project mavenproject1: Execution default-cli of goal fish.payara.maven.plugins:payara-micro-maven-plugin:1.0.0-SNAPSHOT:start failed: Errors occurred while executing payara-micro.: NullPointerException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal fish.payara.maven.plugins:payara-micro-maven-plugin:1.0.0-SNAPSHOT:start (default-cli) on project mavenproject1: Execution default-cli of goal fish.payara.maven.plugins:payara-micro-maven-plugin:1.0.0-SNAPSHOT:start failed: Errors occurred while executing payara-micro.
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:213)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:154)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:146)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
        at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
        at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:309)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:194)
        at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:107)
        at org.apache.maven.cli.MavenCli.execute(MavenCli.java:993)
        at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:345)
        at org.apache.maven.cli.MavenCli.main(MavenCli.java:191)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
        at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
        at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution default-cli of goal fish.payara.maven.plugins:payara-micro-maven-plugin:1.0.0-SNAPSHOT:start failed: Errors occurred while executing payara-micro.
        at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:145)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
        ... 20 more
Caused by: java.lang.RuntimeException: Errors occurred while executing payara-micro.
        at fish.payara.maven.plugins.micro.StartMojo$1.run(StartMojo.java:144)
        at java.lang.Thread.run(Thread.java:748)
        at fish.payara.maven.plugins.micro.StartMojo.execute(StartMojo.java:156)
        at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
        ... 21 more
Caused by: java.lang.NullPointerException
        at java.lang.ProcessBuilder.start(ProcessBuilder.java:1012)
        at java.lang.Runtime.exec(Runtime.java:620)
        at java.lang.Runtime.exec(Runtime.java:528)
        at fish.payara.maven.plugins.micro.StartMojo$1.run(StartMojo.java:132)
        ... 24 more
[ERROR]
```

Interestingly, manually running the created jar (`java -jar target\mavenproject1-1.0-SNAPSHOT-microbundle.jar`) works as expected.

Tested on Windows 10 with latest JDK 8 and Payara Micro 4.1.2.172.

[example.zip](https://github.com/payara/maven-plugins/files/1132028/example.zip)

--- a/payara-micro-maven-plugin/src/main/java/fish/payara/maven/plugins/micro/StartMojo.java
+++ b/payara-micro-maven-plugin/src/main/java/fish/payara/maven/plugins/micro/StartMojo.java
@@ -52,6 +52,8 @@ import org.apache.maven.plugins.dependency.fromConfiguration.ArtifactItem;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.List;
 
@@ -157,9 +159,15 @@ public class StartMojo extends BasePayaraMojo {
         }
     }
 
-    private String decideOnWhichMicroToUse() {
+    private String decideOnWhichMicroToUse() throws MojoExecutionException {
         if (useUberJar) {
-            return evaluateProjectArtifactAbsolutePath(true);
+            String path = evaluateProjectArtifactAbsolutePath(true);
+
+            if (!Files.exists(Paths.get(path))) {
+                throw new MojoExecutionException("\"useUberJar\" option was set to \"true\" but detected path " + path + " does not exist. You need to execute the \"bundle\" goal before using this option.");
+            }
+
+            return path;
         }
 
         if (payaraMicroAbsolutePath != null) {
@@ -178,7 +186,7 @@ public class StartMojo extends BasePayaraMojo {
             return payaraMicroArtifact.getFile().getAbsolutePath();
         }
 
-        return null;
+        throw new MojoExecutionException("Could not determine Payara Micro path. Please set it by defining either \"useUberJar\", \"payaraMicroAbsolutePath\" or \"artifactItem\" configuration options.");
     }
 
     private String evaluateProjectArtifactAbsolutePath(Boolean withExtension) {

